common/meta/grpc/src/grpc_client.rs

```diff
@@ -775,10 +878,24 @@ impl MetaGrpcClient {
 
         Ok(reply)
     }
+    async fn mark_as_unhealthy(&self) {
+        {
+            let ca = self.current_endpoint.lock();
+            let mut ue = self.unhealthy_endpoints.lock();
+            ue.insert((*ca).as_ref().unwrap().clone(), ());
+        }
+        {
+            let mut token = self.token.write().await;
+            *token = None;
+        }
```

Comment:
There is a concurrency issue that before 888, another task created a client, then the current task reset the token to None, which belongs to another connection.